.github/CONTRIBUTING.md

```diff
@@ -395,7 +395,7 @@ Once you sign a CLA, all your existing and future pull requests will have the st
 [up-for-grabs]: https://github.com/powershell/powershell/issues?q=is%3Aopen+is%3Aissue+label%3AUp-for-Grabs
 [semantic linefeeds]: https://rhodesmill.org/brandon/2012/one-sentence-per-line/
 [PowerShell-Docs]: https://github.com/powershell/powershell-docs/
-[use-vscode-editor]: ../docs/learning-powershell/using-vscode.md#editing-with-visual-studio-code
+[use-vscode-editor]: https://docs.microsoft.com/powershell/scripting/components/vscode/using-vscode?view=powershell-6
```

Comment:
@stevend811 The docs is for PowerShell _script_ development. We need link to using VS Code for C# development.

Reply:
@iSazonov All right, I had chosen that link based on the broken link previously leading to a [page on script development](https://github.com/PowerShell/PowerShell/blob/7076d806314e12a1b9ddef2a90d6108ddb7f50b6/docs/learning-powershell/using-vscode.md). Would https://docs.microsoft.com/dotnet/core/tutorials/with-visual-studio-code be a good choice here?

Reply:
The new link looks more appropriated.